packages.ini

```diff
@@ -1177,6 +1179,8 @@ brew_requires = libmemcached
 
 [pyproject-hooks==1.0.0]
 
+[pyright==1.1.372]
+
```

Comment:
isn't this an npm package? why are we installing it python?

plus the standard type checking tool at sentry is mypy, not pyright

Reply:
It was added in this PR [here](https://github.com/getsentry/ops/commit/b67c08af94ba687f7cb1b3d2064b6692b4167132#diff-421cbf24c0e5ea91078eaf7316a03fff4c69a3e04cc2e4d98a4ec8099eb36538R7). @fpacifici do you have context on why pyright is needed? I do not have context to answer the question Anthony raised.

P.S. I am currently working on building the mypy checks in the sentry-infra-tools repo

Reply:
It seems the existing [GHA](https://github.com/getsentry/ops/blob/master/.github/workflows/type-checking.yaml) for type checking indeed only uses mypy. I have a hunch that pyright might not be needed.

Reply:
pyright does not seem to be needed as can be seen [here](https://github.com/getsentry/sentry-infra-tools/pull/24/checks?sha=105ac6e629e4cb18587e3ce0eb33c333c3acfdf0). Removed adding that dependency from the PR.